frontend/public/components/service.jsx

```diff
@@ -22,7 +22,7 @@ const menuActions = [Cog.factory.ModifyPodSelector, ...Cog.factory.common];
 const ServiceIP = ({s}) => {
   const children = _.map(s.spec.ports, (portObj, i) => {
     const clusterIP = s.spec.clusterIP === 'None' ? 'None' : `${s.spec.clusterIP}:${portObj.port}`;
-    return <span key={i}>{clusterIP}&nbsp;&nbsp;</span>;
+    return <div key={i}>{clusterIP}&nbsp;&nbsp;</div>;
```

Comment:
I think we can remove the non breaking spaces now

Reply:
removed them